.github/pr-custom-review.yml

```diff
@@ -0,0 +1,19 @@
+action-review-team: opstooling
+
+rules:
+  - name: Core developers
+    check_type: changed_files
+    condition:
+      include: .*
+    min_approvals: 2
+    teams:
+      - opstooling
+
+  - name: CI files
+    check_type: changed_files
+    condition:
+      # dictionary files are excluded
+      include: ^\.gitlab-ci\.yml
```

Comment:
I also think that `/helm` directory should require CI review as well

Reply:
Added to the check. Can you review it again please?